src/main/java/by/bcrypto/bee2j/der/DerBitString.java

```diff
@@ -0,0 +1,57 @@
+package by.bcrypto.bee2j.der;
+
+import java.io.IOException;
+import com.sun.jna.ptr.LongByReference;
+
+import by.bcrypto.bee2j.Bee2Library;
+
+public class DerBitString extends DerValue {
+
+    private byte[] bits;
+    private int bitLength;
+
+    DerBitString(byte[] der, byte tag, long length, long offset) {
```

Comment:
Let's add access modifiers for constructors and methods to keep the code consistent. 
Do it pls for `DerSequence` and `DerValue` too.

Reply:
These methods were planned to be package-level accessed, but it is not problem for me to make them public. Fixed in 4ae65a024ec0b87e172bca84b932004e5843f5ca